_layouts/default.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: 'en-US' }}" class="no-js">
+    
+    <head>
+
+        <title>{{ site.title }}</title>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <script async defer src="https://cdn.simpleanalytics.io/hello.js"></script>
+        <noscript><img src="https://api.simpleanalytics.io/hello.gif" alt=""></noscript>
+      
+        <!-- Twitter cards -->
+        <meta name="twitter:site"    content="@{{ site.twitter_username }}">
+        <meta name="twitter:creator" content="@{{ page.author }}">
+        <meta name="twitter:title"   content="{{ page.title }}">
+      
+        {% if page.summary %}
+        <meta name="twitter:description" content="{{ page.summary }}">
+        {% else %}
+        <meta name="twitter:description" content="{{ site.description }}">
+        {% endif %}
+      
+        {% if page.image %}
+        <meta name="twitter:card"  content="summary_large_image">
+        <meta name="twitter:image" content="{{ site.url }}{{ site.baseurl }}{{ page.image }}">
+        {% else %}
+        <meta name="twitter:card"  content="summary">
+        <meta name="twitter:image" content="{{ site.title_image }}">
+        {% endif %}
+        <!-- end of Twitter cards -->
+
+        <link href="https://fonts.googleapis.com/css?family=Lexend+Deca&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.5.1/css/swiper.min.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <link rel="stylesheet" href="{{ '/assets/css/main.css' | relative_url }}">
+    </head>
+
+  <body class="layout--{{ page.layout | default: layout.layout }}{% if page.classes or layout.classes %}{{ page.classes | default: layout.classes | join: ' ' | prepend: ' ' }}{% endif %} {{ page.title | slugify }}">
+
+    <div class="main-container">
+
+        {% include header.html %}
+        {{ content }}
+        {% include footer.html %}
+
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
```

Comment:
What are we using jquery and Swiper for out of curiosity? If we can minimize page load that'd be good

Reply:
Same with google fonts - do we 100% need them? For logo 100%, but I'm not sure if we need custom fonts for body text (e.g. in case of individual posts design it looks less readable to me)

Reply:
We are using Swiper and jQuery for the sliders on mobile. 

We need the fonts too since those 2 are used in the design.